FirebaseAppCheck.podspec

```diff
@@ -45,7 +45,7 @@ Pod::Spec.new do |s|
   s.osx.weak_framework = 'DeviceCheck'
   s.tvos.weak_framework = 'DeviceCheck'
 
-  s.dependency 'AppCheckCore', '~> 10.13'
+  s.dependency 'AppCheckCore', '~> 0.1.0-alpha'
```

Comment:
There should be a TODO before this and next change to undo before merge to master.

Reply:
Good call! Added a TODO here and in `ReleaseTooling/Sources/PodspecsTester/main.swift`.